From c95263d877ca3ed953de0a114b2c1ccb9aa04b65 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 4 Apr 2017 17:55:50 +0200
Subject: [PATCH 1/2] remove explicit `|| return 1` from all sample APKBUILDs

Those statement are no longer needed because abuild now runs with
`set -e` by default.

See also: 36d5193776180385a39626a83241822736a5f6b8
---
 newapkbuild.in  | 28 +++++++++++++---------------
 sample.APKBUILD |  6 +++---
 2 files changed, 16 insertions(+), 18 deletions(-)

diff --git a/newapkbuild.in b/newapkbuild.in
index c021db5..bbe7a44 100644
--- a/newapkbuild.in
+++ b/newapkbuild.in
@@ -33,7 +33,7 @@ is_github_url() {
 # Build sections
 build_make() {
 	cat >>APKBUILD<<__EOF__
-	make || return 1
+	make
 __EOF__
 }
 
@@ -45,9 +45,8 @@ build_autotools() {
 		--prefix=/usr \\
 		--sysconfdir=/etc \\
 		--mandir=/usr/share/man \\
-		--localstatedir=/var \\
-		|| return 1
-	make || return 1
+		--localstatedir=/var
+	make
 __EOF__
 }
 
@@ -71,29 +70,28 @@ build_cmake() {
 		-DCMAKE_BUILD_TYPE=Release \\
 		-DCMAKE_CXX_FLAGS="\$CXXFLAGS" \\
 		-DCMAKE_C_FLAGS="\$CFLAGS" \\
-		${CMAKE_CROSSOPTS} \\
-		|| return 1
-	make || return 1
+		${CMAKE_CROSSOPTS}
+	make
 __EOF__
 }
 
 build_perl() {
 	cat >>APKBUILD<<__EOF__
-	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor || return 1
-	make || return 1
+	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor
+	make
 __EOF__
 }
 
 build_python() {
 	cat >>APKBUILD<<__EOF__
-	python setup.py build || return 1
+	python setup.py build
 __EOF__
 }
 
 # Package sections
 package_make() {
 	cat >>APKBUILD<<__EOF__
-	make DESTDIR="\$pkgdir" install || return 1
+	make DESTDIR="\$pkgdir" install
 __EOF__
 }
 
@@ -103,14 +101,14 @@ package_autotools() {
 
 package_perl() {
 	cat >>APKBUILD<<__EOF__
-	make DESTDIR="\$pkgdir" install || return 1
+	make DESTDIR="\$pkgdir" install
 	find "\$pkgdir" \\( -name perllocal.pod -o -name .packlist \\) -delete
 __EOF__
 }
 
 package_python() {
 	cat >>APKBUILD<<__EOF__
-	python setup.py install --prefix=/usr --root="\$pkgdir" || return 1
+	python setup.py install --prefix=/usr --root="\$pkgdir"
 __EOF__
 }
 
@@ -273,9 +271,9 @@ __EOF__
 		cat >>APKBUILD<<__EOF__
 
 	install -m755 -D "\$srcdir"/\$pkgname.initd \\
-		"\$pkgdir"/etc/init.d/\$pkgname || return 1
+		"\$pkgdir"/etc/init.d/\$pkgname
 	install -m644 -D "\$srcdir"/\$pkgname.confd \\
-		"\$pkgdir"/etc/conf.d/\$pkgname || return 1
+		"\$pkgdir"/etc/conf.d/\$pkgname
 __EOF__
 	fi
 	cat >>APKBUILD<<__EOF__
diff --git a/sample.APKBUILD b/sample.APKBUILD
index 30184c5..95a6634 100644
--- a/sample.APKBUILD
+++ b/sample.APKBUILD
@@ -31,12 +31,12 @@ build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info
-	make || return 1
+	make
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 
 	# remove the 2 lines below (and this) if there is no init.d script
 	# install -m755 -D "$srcdir"/$pkgname.initd "$pkgdir"/etc/init.d/$pkgname
@@ -47,7 +47,7 @@ check() {
 	# uncomment the 2 lines below if there is a testsuite.  we assume the testsuite
 	# is run using "make check", which is the default for autotools-based build systems.
 	# cd "$builddir"
-	# make check || return 1
+	# make check
 }
 
 md5sums="" #generate with 'abuild checksum'
-- 
2.12.2

